src/data/methods/Ripple.methods.ts

```diff
@@ -18,3 +18,12 @@ export type RIPPLE_REQUESTS = {
     submit?: boolean;
   };
 };
+
+/**
+ * Responses specs: https://docs.reown.com/advanced/multichain/rpc-reference/xrpl-rpc
+ */
+export type RIPPLE_RESPONSES = {
+  [RIPPLE_SIGNING_METHODS.RIPPLE_SIGN_TRANSACTION]: {
+    tx_json: Record<string, unknown>;
```

Comment:
I think we can type it like this
```suggestion
    tx_json: XrpTransaction;
```

Reply:
Yes! Also, we had to add the transaction hash:
`tx_json: XrpTransaction & { hash: string };`
And now __it works great!__ I tested it.